cli/src/diff_util.rs

```diff
@@ -296,6 +297,7 @@ impl<'a> DiffRenderer<'a> {
         matcher: &dyn Matcher,
         copy_records: &CopyRecords,
         width: usize,
+        conflict_marker_style: ConflictMarkerStyle,
```

Comment:
nit: Perhaps, the style can be captured by `DiffRenderer::new()` for now.

If we add native support for conflict diffs, the `conflict_marker_style` might be moved to `DiffFormat` variants. And `[diff.color-words]` might have `materialize-conflict = true|false` and `conflict-marker-style = <style>`, for example.